voice-client-javascript/tests/admin/login.test.js

```diff
@@ -0,0 +1,67 @@
+const path = require("path");
+const helpers = require("../../../test/test-helper");
+const baseDir = path.resolve("./voice-client-javascript/");
+
+// Defined after Runtime is available
+let loginFunction;
+let createToken;
+
+describe("voice-client-javascript/admin/login", () => {
+  beforeAll(() => {
+    helpers.setup({
+      baseDir,
+    });
+    process.env.ADMIN_PASSWORD = "supersekret";
+    return Runtime.load().then(() => {
```

Comment:
@dkundel @philnash It's a little clunky, but couldn't figure away around the problem of requiring a file that requires assets. Thoughts?

Reply:
I do suppose if we require the `getAssets()` call to require using the `context.PATH` it can only be loaded dynamically which would make this safe.

Reply:
Sorry that we are sidetracking you with this :) I think we can make this a bit better though. I'd like to keep the amount of work with the filesystem and logic in the test-helpers to a minimum but at the same time I understand that this is a requirement for your tests. 
I would prefer if we set the available assets more explicitly. Maybe the right API would be something like:

```js
const context = {}
const availableAssets = {
'/admin/shared.js': path.resolve(__dirname, '../../assets/admin/shared.private.js')
}
const availableFunctions = {}
helpers.setup(context, { assets: availableAssets, functions: availableFunctions })
```

@philnash do you have an idea?

Reply:
I like the explicit call to what's available. Most of the test setup in these tests has been deliberately explicit and this would go along nicely with that.

Reply:
No worries on the sidetracking. Glad to get this sorted as it's bound to pop up as these function-templates grow.

Wondering if that second param should actually be a `MockRuntime` instance. Would help with the knowledge, and maybe avoid the `path` import. 

```javascript
const runtime = new MockRuntime();
runtime._addAsset('/admin/shared.js', '../../assets/admin/shared.private.js');
helpers.setup(context, runtime);
```

This would also allow you as a test developer to craft your own.

Reply:
Yes I think that's a great solution!